src_c/rect.c

```diff
@@ -37,6 +37,27 @@
 static PyTypeObject pgRect_Type;
 #define pgRect_Check(x) ((x)->ob_type == &pgRect_Type)
 
+#define PG_RETURN_TUPLE_FROMVALUES_INT(val1, val2) \
```

Comment:
I wonder if there is a speed difference if this is a function?

How's the generated code size?

Reply:
> I wonder if there is a speed difference if this is a function?

Apparently not, just tested.

> How's the generated code size?

This I didn't test

Reply:
> Apparently not, just tested.

Cool